stats/Record.md

```diff
@@ -67,6 +69,7 @@ private static final MeasureLong RPC_BYTES_SENT =
 MeasurementMap measurementMap = new MeasurementMap();
 measurementMap.put(RPC_LATENCY, 10.3);
 measurementMap.put(RPC_BYTES_SENT, 124);
+measurementMap.withSpanContext(spanContext);
```

Comment:
Should the Java implementation support using the current span?  In that case, I think the no-arg `record` method should  add the spanContext implicitly.

Reply:
Do you mean adding another no-arg `recordWithCurrentSpanContext()` method? I think for the existing no-arg `record()` won't interact with spanContext, to be consistent with previous behavior?

Reply:
What is the harm in sending exemplars by default? Why do we need to remain consistent with the previous behavior?

If we do, I would recommend instead that we configure exemplar support on the exporter (those that support it, which is just Stackdriver for now). 

I feel quite strongly that we shouldn't require people to change their instrumentation to get exemplars if we can avoid it.

Reply:
I don't object to make it default to record again current span context, though I'm wondering will there be cases that user would only want histograms, but not linked to trace spans? If so, I think we should at least provide a method that records stats against no span context.